front/app/components/CustomFieldsForm/IdeationForm/index.tsx

```diff
@@ -76,10 +76,14 @@ const IdeationForm = ({
         updateSearchParams({ idea_id: idea.data.id });
         trackEventByName(tracks.ideaFormSubmitted);
       } else {
+        // Strip away idea_files_attributes from the form values
+        // as they are handled via separate API calls
+        const { idea_files_attributes: _, ...formValuesWithoutFiles } =
```

Comment:
This is the new fix, we simply strap away the `idea_files_attributes` from the api request. The other approach was creating validation problems when the files field was required.